modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -170,14 +176,19 @@ public DistributionZoneManager(
      * @return Future representing pending completion of the operation. Future can be completed with:
      *      {@link DistributionZoneAlreadyExistsException} if a zone with the given name already exists,
      *      {@link ConfigurationValidationException} if {@code distributionZoneCfg} is broken,
-     *      {@link IllegalArgumentException} if distribution zone configuration is null,
+     *      {@link IllegalArgumentException} if distribution zone configuration is null
+     *      or distribution zone name is {@code DEFAULT_ZONE_NAME},
      *      {@link NodeStoppingException} if the node is stopping.
      */
     public CompletableFuture<Void> createZone(DistributionZoneConfigurationParameters distributionZoneCfg) {
         if (distributionZoneCfg == null) {
             return failedFuture(new IllegalArgumentException("Distribution zone configuration is null"));
         }
 
+        if (DEFAULT_ZONE_NAME.equals(distributionZoneCfg.name())) {
+            return failedFuture(new IllegalArgumentException("Default distribution zone cannot be recreated."));
```

Comment:
I'd rather use following message: 
"It's not possible to create distribution zone with [name= " + DEFAULT_ZONE_NAME + ']'

Reply:
fixed